src/Framework/LazyFormattedBuildEventArgs.cs

```diff
@@ -179,6 +180,8 @@ internal override void CreateFromStream(BinaryReader reader, Int32 version)
             }
         }
 
+        internal string LogOutputProperties{ get;  set; }
```

Comment:
I'm not a fan of mutating the event object mid-logging. It looks like it would be possible to keep this in the logger itself and pass it to formatting calls, right? Could we do that instead of adding a mutable field to a public type, even if the field is internal?

Reply:
Ya we can. I don't want to change the paramters of the current format methods because that could cause potential breaks elsewhere but I can add another overload method that passes it.